tokens/font.json

```diff
@@ -3,27 +3,42 @@
     "type": "size",
     "category": "font-size"
   },
-  "props": {
-    "FONT_SIZE_TINY": {
-      "value": "0.625rem"
+  "props": [
+    {
+      "name": "FONT_SIZE_TINY",
+      "value": "0.64rem"
     },
-    "FONT_SIZE_SMALL": {
-      "value": "0.875rem"
+    {
+      "name": "FONT_SIZE_SMALL",
+      "value": "0.8rem"
     },
-    "FONT_SIZE_NORMAL": {
+    {
+      "name": "FONT_SIZE_ROOT",
+      "value": "16px"
+    },
+    {
+      "name": "FONT_SIZE_NORMAL",
       "value": "1rem"
     },
-    "FONT_SIZE_MEDIUM": {
+    {
+      "name": "FONT_SIZE_MEDIUM",
       "value": "1.25rem"
     },
-    "FONT_SIZE_LARGE": {
+    {
+      "name": "FONT_SIZE_LARGE",
       "value": "1.5rem"
     },
-    "FONT_SIZE_X_LARGE": {
-      "value": "2.5rem"
+    {
+      "name": "FONT_SIZE_X_LARGE",
+      "value": "1.953rem"
+    },
+    {
+      "name": "FONT_SIZE_XX_LARGE",
+      "value": "2.441rem"
```

Comment:
**Checking my assumptions**
If our base font-size is 16px, this would be `16px*2.441=39.05600px`, right?

**My baseline unit assumption**
And if we apply a line-height of 1.5 to this, our base unit would be 24px.

**My typographic scale conclusion?**
But the line height of this typographic style would be `39.05600px*1.5=58.58400px`, which does not adhere to our typographic scale?

Reply:
> If our base font-size is 16px, this would be 16px*2.441=39.05600px, right?

Yes, which is rounded to 39.06px on https://type-scale.com/.

> And if we apply a line-height of 1.5 to this, our base unit would be 24px.

Correct.

> But the line height of this typographic style would be 39.05600px*1.5=58.58400px, which does not adhere to our typographic scale?

No, the computed line-height of all text elements should be a multiple of 24px. This is a heading font, so we would need to tweak the ratio. In the ["Varying Repetitions" section](https://zellwk.com/blog/why-vertical-rhythms/), it states that the ratio should be adjusted so that the new computed line height for the header is a multiple of 24.

Reply:
Ahh, gotcha.

- We'll scale the font-sizes with a major third scale. I'd be fine with this.
- We'll manually add line-heights to match the vertical Rythm.
- In the articles case, they suggest always using a multiple of 12px (12px, 24px, 36px, 48px, 60px, 72px …)

These 12 pixel jumps are slightly too big for smaller font-sizes. Google's Material Design uses a 4px baseline grid for typography and text, and an 8px baseline grid for layout other things. Would this work for you?

_If we choose this approach, we'll be keeping a vertical rhythm, but the line-heights for different headings will be a different ratio. I've set up a Google sheet in which I round the line-heights to the closest value of 4px, and even there, the line-height would vary between ~1.15 and ~1.25 if we use the base-ratio of 1.2 for headlines._

Reply:
> These 12 pixel jumps are slightly too big for smaller font-sizes. Google's Material Design uses a 4px baseline grid for typography and text, and an 8px baseline grid for layout other things to solve this. Would this work for you?

For this, you could use a ratio of 0.5, which is completely fine.  👍

Reply:
As small as 0.5 is 12px, which would mean that a 12px text somewhere would either be 12px `line-height: 1` or 24px `line-height: 2`

Reply:
I was just correcting my post! 🦆 Indeed, you're absolutely right.